Ustring.php

```diff
@@ -220,6 +212,26 @@ public function __construct($string = null)
     }
 
     /**
+     * Check if ext/mbstring is available.
+     *
+     * @return  bool
+     */
+    public static function checkMbString()
+    {
+        return function_exists('mb_substr');
```

Comment:
No because we allow third implentation, such as https://github.com/nicolas-grekas/Patchwork-UTF8 (ping @nicolas-grekas).